src/components/Content/Card.js

```diff
@@ -1,27 +1,83 @@
 // Card.js
 import React from 'react';
-import { AddButtonContainer, CardContainer, CardDetails } from './Card.styles';
+import {
+  AddButtonContainer,
+  CardContainer,
+  CardDetails,
+  DateTimeContainer,
+  TopDetailsContainer,
+  BottomDetailsContainer,
+} from './Card.styles';
 import { CardFooter } from './Card.styles';
 import { Text } from '../Text/Text';
 import { faTimes, faTrash } from '@fortawesome/free-solid-svg-icons';
-import Icon from '../Icon/Icon'; // Import the corrected Icon component
 import { darkModeColors } from '../../constants/colors';
 import { Footer } from '../Footer/Footer';
+
+import CalendarDark from '../../Assets/dark/calendar.png';
+import LocationDark from '../../Assets/dark/location.png';
+import { Image } from '../Image/Image';
+
+//create constant for resued sizes
+const SemiBold = '600';
```

Comment:
Could create a separate constants file if these sizes are also being used in other components

Reply:
Done :)